### ISSUE TITLE
Mind Implant Case Unlocked when showing Locked

### ISSUE THREAD

**Issue by Davidj361** [tgstation/tgstation#37086]
[Round ID]: 86296
86296

[Reproduction]:
1. Go to the armory
2. Pickup the mind implant case
3. The mind implant case should have a red light on it indicating it's locked
4. Be able to access the briefcase anyways

**Comment by KorPhaeron** [comment]
Which map?

**Comment by Davidj361** [comment]
I think box station? Though, I think this is a problem on all maps.

**Comment by iskyp** [comment]
yeah, i can confirm this happened to me as well. Picked up the mind implant case, clearly locked, but still accessed it.
